controller/konnect/reconciler_generic_test.go

```diff
@@ -36,21 +95,53 @@ func testNewKonnectEntityReconciler[
 ](
 	t *testing.T,
 	ent T,
+	testCases []konnectEntityReconcilerTestCase,
 ) {
 	t.Helper()
 
-	// TODO: use a mock Konnect SDK factory here and use envtest to trigger real reconciliations and Konnect requests
 	sdkFactory := &ops.MockSDKFactory{}
 
 	t.Run(ent.GetTypeName(), func(t *testing.T) {
-		cl := fakectrlruntimeclient.NewFakeClient()
-		mgr, err := ctrl.NewManager(&rest.Config{}, ctrl.Options{
+		s := scheme.Get()
+		require.NoError(t, configurationv1alpha1.AddToScheme(s))
+		require.NoError(t, configurationv1beta1.AddToScheme(s))
+		require.NoError(t, konnectv1alpha1.AddToScheme(s))
```

Comment:
The scheme that we get via `scheme.Get()` already has all those added to it, no? https://github.com/Kong/gateway-operator/blob/d5f3d3672f8eba3b82185c39a5b2c993bb6ca191/modules/manager/scheme/scheme.go#L20-L37

Reply:
Removed these.